.github/workflows/ci.yml

```
@@ -17,12 +17,14 @@ jobs:
       - uses: actions/checkout@v2
       - uses: conda-incubator/setup-miniconda@v2
         with:
+          auto-update-conda: true
```

```suggestion
          miniforge-variant: Mambaforge
          use-mamba: true
```
Curious, is this faster? And that way you can drop the conda config calls below.